docs/dev-guide.md

```diff
@@ -196,9 +198,11 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 ## DB schema (simplified overview)
 
-- The most important tables in the database are shown in the diagram below (also as a [PDF image](simplified-models.pdf)).
+- The most important tables in the database are shown in the diagram below
 
-![Simplified DB Schema](simplified-models.png)
+```d2
+--8<-- "docs/diagrams/simplified-models.d2"
+```
 
 - Detailed descriptions of the format of each table are in the [list of DB tables](https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/).
```

Comment:
Tiny nit, I think spelling out "database" is nicer (if you like this, this can be changed elsewhere in the file):

```suggestion
- Detailed descriptions of the format of each table are in the [list of database tables](https://github.com/PrairieLearn/PrairieLearn/blob/master/database/tables/).
```